ops/ccs-ops-misc/synthetic-data/scripts/synthea-automation/prepare-and-run-synthea.py

```diff
@@ -59,7 +61,7 @@ def validate_and_run(args):
     if not synthea_folder_filepath.endswith('/'):
         synthea_folder_filepath = synthea_folder_filepath + "/"
     generated_benes = args[2]
-    db_string = args[3]
+    db_data = args[3].split(',')
```

Comment:
I don't know if there's a guarantee that passwords can't contain commas, especially when they're autogenerated. Breaking these out into six separate arguments (or maybe pass them as environment variables? Get them from SSM? 🤷 ) would avoid any problems with delimiters in passwords.

Reply:
Think this might be something I come back to after cmac and I strategize a bit; might end up switching to SSM if its easier than passing credentials in, but I need to figure out what the plan is. For now this seems like it meets our needs. (If I end up swapping to SSM, i'd do it for both this and the other scripts together, since they all need DB access)